youtube_dl/downloader/__init__.py

```diff
@@ -30,14 +32,17 @@ def get_suitable_downloader(info_dict, params={}):
     protocol = determine_protocol(info_dict)
     info_dict['protocol'] = protocol
 
+    if (info_dict.get('start_time') or info_dict.get('end_time')) and FFmpegFD.can_download(info_dict):
```

Comment:
This should go after the `external_downloader` check. Now when `--external-downlaoder` is specified and there is a [start; end] range ffmpeg is used anyway.